### PR TITLE
fix: allow landing page scroll

### PR DESCRIPTION
**Pull request by vm**
Avoids the state for now where on mobile the button isn't clickable

### CODE REVIEW

File: src/pages/Landing/index.tsx [hunk 1]
```diff
@@ -172,18 +171,6 @@ export default function Landing() {
 
   const landingPageFlag = useLandingPageFlag()
```
Comment (just-toby) [inline src/pages/Landing/index.tsx:172]: should we remove this now?

**Review by yyip-dev — APPROVED** [review]

**Review by just-toby — APPROVED** [review]
🛳️

File: src/pages/Landing/index.tsx (hunk 1 above)
Comment (vm) [reply src/pages/Landing/index.tsx:172]: i have a pr for it

Comment (vm) [reply src/pages/Landing/index.tsx:172]: this default to enabled for now, and i have a diff pr to remove completely

**Pull request merged**